gems/pending/openstack/openstack_handle/handled_list.rb

```diff
@@ -0,0 +1,118 @@
+module OpenstackHandle
+  module HandledList
+    def handled_list(collection_type, options = {})
+      # Alias with hardcoded :all method, just for convenience, since :all method will be used in 99% of the cases
+      handled_list_method(collection_type, options, :all)
+    end
+
+    def handled_list_method(collection_type, options = {}, method = :all)
```

Comment:
Delete it for now.  If we need to enable that sort of functionality later, we can reintroduce it then.